product_listings_manager/products.py

```diff
@@ -453,18 +453,15 @@ def getModuleProductListings(productLabel, moduleNVR):
         module_name = module['name']
         module_stream = module['stream']
         module_version = module['version']
-    except KeyError:
-        raise koji.GenericError("It's not a module build: %s" % moduleNVR)
+    except (KeyError, TypeError):
+        raise ProductListingsNotFoundError("It's not a module build: %s" % moduleNVR)
 
     prodinfo = Products.get_product_info(compose_dbh, productLabel)
-    if not prodinfo:
-        # no product with the given label exists
-        raise koji.GenericError("Could not find a product with label: %s" % productLabel)
```

Comment:
@hlin Why was this removed? I think the line below would fail if the type `prodinfo` is wrong or it doesn't have exactly two items.

Reply:
It's covered in get_product_info() https://github.com/release-engineering/product-listings-manager/blob/master/product_listings_manager/products.py#L82-L96

Reply:
Oh, OK. Thanks for the new release!